orttraining/orttraining/python/training/_utils.py

```diff
@@ -2,28 +2,11 @@
 import numpy as np
 import os
 import sys
-import time
 import torch
 from onnx import TensorProto
 
 from functools import wraps
 
-def timeit(enabled=True):
```

Comment:
Many of the ```get_device*``` functions are not used in ortmodule. Maybe, we can remove some of them?